### ISSUE TITLE
can't retrieve images from eu.gcr.io

### ISSUE THREAD

**Issue by ToGoBananas** [upmc-enterprises/registry-creds#49]
It works fine with gcr.io images but it fails for eu subdomain

'''rpc error: code = 2 desc = Error response from daemon: denied: Permission denied for "latest" from request'''

**Comment by stevesloka** [comment]
Yeah looks like I need to better support that domain. Right now it assumes the `gcr.io`. You can override that in the meantime by passing an arg to the process. See: https://github.com/upmc-enterprises/registry-creds/blob/master/main.go#L62

**Comment by vongohren** [comment]
Where are these args to be inputted?

**Comment by stevesloka** [comment]
Hey @vongohren, you add that to the args of the controller definition.

**Comment by vongohren** [comment]
But if we use this as an addon to minikube, we dont have access to the controller?

**Comment by Teddy-Schmitz** [comment]
you can find the controller in the kube-system namespace when using minikube
